website/www/site/content/en/blog/apache-beam-flink-and-kubernetes-part3.md

```diff
@@ -1,6 +1,7 @@
 ---
 title:  "Behind the Scenes: Crafting an Autoscaler for Apache Beam in a High-Volume Streaming Environment"
 date:   2023-12-21 09:00:00 -0400
+math: true
```

Comment:
Looks like this did not work. Could you try writing the equations in ways that don't require this special math block?